lib/msf/core/db_manager/host.rb

```diff
@@ -157,8 +157,6 @@ def report_host(opts)
 
     if not addr.kind_of? ::Mdm::Host
       addr = normalize_host(addr)
-      addr, scope = addr.split('%', 2)
-      opts[:scope] = scope if scope
```

Comment:
This looks wrong.  Did IPv6 handling change in the database that this is no longer needed?